### ISSUE TITLE
How to preview help_tags?

### ISSUE THREAD

**Issue by uga-rosa** [ibhagwan/fzf-lua#120]
The built-in previewer doesn't work. If anyone is doing this, please let me know.

**Comment by ibhagwan** [comment]
> The built-in previewer doesn't work. If anyone is doing this, please let me know.

Wasn't implemented yet.

**Comment by uga-rosa** [comment]
Thanks for the quick reply! I will wait for the implementation.

**Comment by ibhagwan** [comment]
Preview with syntax was added for both `help_tags` and `man_pages`, update your plugin and it should work.

**Comment by uga-rosa** [comment]
Thank you!
Is it possible to turn off one line buffer(?) at the top of the screen?

![image](https://user-images.githubusercontent.com/82267684/132664224-a49373b9-ce00-46cb-b34d-fd67ffd931dd.png)

**Comment by ibhagwan** [comment]
Not until I figure out how to send the `help` command directly to a buffer, I use `:help` to locate the right help file and tag but turns out neovim doesn’t let you control where it opens so for now I had to minimize that buffer to 1 line and make it look like a “functionality” lol

**Comment by uga-rosa** [comment]
There may be a hint in telescope.nvim, but it's hard to read that code lol
In fact, it's not much of a problem.

**Comment by kdheepak** [comment]
The help tag preview is an awesome feature! Thanks for all your work on this!

**Comment by ibhagwan** [comment]
> There may be a hint in telescope.nvim, but it's hard to read that code lol
> In fact, it's not much of a problem.

I read it already, Telescope uses a different method and reads the help file directly and then searches for the help tag pattern inside it (that's why the help term is highlighted as if you did `/<query>`).

I'm just testing some more stuff, going to make sure the bar stays in the bottom so it's less annoying, you probably don't have `splitbelow` set that's why it opens in the top, I'll update this this therad when I commit the change soon.

**Comment by uga-rosa** [comment]
Hmm. If it's down, the status bar collapses. Personally, I think that appearing on the right side is the least obtrusive. It would be nice to have a choice.

**Comment by ibhagwan** [comment]
> Hmm. If it's down, the status bar collapses. Personally, I think that appearing on the right side is the least obtrusive. It would be nice to have a choice.

Made some changes to make this work in fzf-lua split mode too (not popup), the default bar location was set to bottom, unfortunately right or left is not possible as nvim complains about not enough width with the window width is set to 1.

If you wish to revert back to top bar just set this in your `setup()`:

```lua
  require'fzf-lua'.setup {
    -- uncomment to set dummy win location (help|man bar)
    -- "topleft"  : up
    -- "botright" : down
    helptags = { previewer = { split = "topleft" } },
    manpages = { previewer = { split = "topleft" } },
  }
```

**Comment by ibhagwan** [comment]
I force pushed the repo due to an extra debug print statement, so if you updated before the force push make sure to delete the plugin folder before updating again, if you didn't update in the last 10 minutes you got nothing to worry about.

**Comment by uga-rosa** [comment]
I see. Thanks.
